### DISCUSSION TITLE
Started getting 'Could not find config file' error from quasar:eslint after upgrade quasar/eslint

### DISCUSSION THREAD

**Discussion by evil-shrike** [quasarframework/quasar#17224] [General - Components / Directives / etc]
Hello.

I've updated packages in my existing project. Particularly:
```
   @quasar/extras": "^1.16.7" 
    "quasar": "^2.13.0",
    "vue": "^3.3.7",
    "vue-router": "^4.2.5",
    "vue3-apexcharts": "^1.4.4",
    "@quasar/app-vite": "^1.6.2",
    "eslint": "^8.10.0",
    "eslint-config-prettier": "^8.1.0",
    "eslint-plugin-vue": "^9.0.0",
    "prettier": "^3.0.3",
```
to
```
   "@quasar/extras": "^1.16.11",
    "quasar": "^2.16.4",
    "vue": "^3.4.27",
    "vue-router": "^4.3.2",
    "vue3-apexcharts": "^1.5.3",
    "@quasar/app-vite": "^1.9.3",
    "eslint": "^9.3.0",
    "eslint-config-prettier": "^9.1.0",
    "eslint-plugin-vue": "^9.26.0",
    "prettier": "^3.2.5",
```

and now I'm getting a build error:
<img width="1476" alt="image" src="https://github.com/quasarframework/quasar/assets/1203230/47e4c323-9f35-4612-92f8-917a0ed3cce8">

﻿Error: Could not find config file. From plugin: 'quasar:eslint'.

Not sure what should I update. And which upgrade caused it. It was building just fine before that.
My `quasar.config.js` contains only this related to eslint:
```
  return {
    eslint: {
      // fix: true,
      // include: [],
      // exclude: [],
      // rawOptions: {},
      warnings: true,
      errors: true,
    },
```

My `.eslintrc.cjs` is in the same folder as quasar.config.js. I never touched it since it was generated by quasar cli.

**Comment by yusufkandemir** [comment]
eslint v9 doesn't support `.eslintrc.cjs`, it has a different config format called flat config. https://eslint.org/docs/latest/use/migrate-to-9.0.0
